src/App.js

```diff
@@ -53,6 +54,7 @@ const App = () => {
         <Route exact path="/students" component={AllStudentsContainer} />
         <Route exact path="/newstudent" component={NewStudentContainer} />
         <Route exact path="/student/:id" component={StudentContainer} />
+        <Route exact path="/student/:firstname/edit" component={EditStudentContainer} />
```

Comment:
Curious why this is /student/:firstname/edit and not /student/:id/edit?

Reply:
Ugh this was a bug. I had the paths setup to display the name instead of the id. I changed the other ones, mustve missed it.